workspace/GoogleMapsUtilities/GoogleMapsUtilities.h

```diff
@@ -0,0 +1,73 @@
+/* Copyright (c) 2017 Google Inc.
```

Comment:
2019*

Reply:
Suggestion. 2020*

Reply:
Ok, Guys, I will update and merge from master and copyright.

Reply:
how long this will need to be released for pods usage ? I'm currently working on a project due tomorrow and I could not integrate without pods